src/vs/workbench/contrib/remote/common/remote.contribution.ts

```diff
@@ -355,6 +359,19 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 				enum: ['localhost', 'allInterfaces'],
 				default: 'localhost',
 				description: localize('remote.localPortHost', "Specifies the local host name that will be used for port forwarding.")
+			},
+			[REMOTE_DEFAULT_EXTENSIONS]: {
+				type: 'array',
+				markdownDescription: localize('remote.defaultExtensions.markdownDescription', 'List of extensions to install automatically on all remotes.'),
+				default: [
+					'GitHub.copilot',
+					'GitHub.copilot-chat'
```

Comment:
I recommend to get these values from the product.json

Reply:
https://github.com/microsoft/vscode/pull/244046